src/styles/blocks/card.scss

```diff
@@ -0,0 +1,134 @@
+$button-color: #00acdc;
+$bg-color: white;
+$smalltext-color: #616070;
+$maintext-color: #060b35;
+
+* {
+  box-sizing: border-box;
+
+}
+
+h1,
+h2,
+p,
+body {
+  margin: 0;
+  padding: 0;
+  font-family: Roboto, Helvetica, sans-serif;
+}
+
+.card {
+  padding: 32px 16px 16px;
+  width: 200px;
+  border-radius: 5px;
+  border: 1px solid #d3d3d3;
+  background-color: $bg-color;
+  display: grid;
+  flex-direction: column;
+  align-items: center;
+
+  &__img {
+    width: 100%;
+    height: 100%;
+    object-fit: contain;
+    display: block;
+    background-image: url("../images/imac.jpeg");
+    background-size: cover;
+    margin-bottom: 0;
+    overflow: hidden;
+  }
+
+  &__title {
+    margin-top: 40px;
+    font-size: 12px;
+    font-weight: 500;
+    line-height: 18px;
+    text-align: left;
+    color: $maintext-color;
+  }
+
+  &__product-code {
+    margin-top: 4px;
+    font-size: 10px;
+    line-height: 14px;
+    text-align: left;
+    color: $smalltext-color;
+    width: 100%;
+  }
+
+  .reviews {
+    display: flex;
+    align-items: center;
+    margin-top: 16px;
+
+    .stars {
```

Comment:
block `stars` is separated and shouldn't be nested, it has own styles that not depending on block review